exts/base.py

```diff
@@ -105,12 +106,18 @@ def __init__(self, bot: commands.Bot):
     @commands.command(name="info", aliases=["about"])
     async def _info(self, ctx: commands.Context):
         """Shows info about qrm."""
+        if path.isfile("git_commit"):
```

Comment:
This could be dine in a neater way with `pathlib`, especially to avoid the path `"git_commit"` happening twice next to variable `git_commit` (thrice present)